src/app/views/editors/ckeditor/ckeditor_preparer.ts

```diff
@@ -21,10 +22,23 @@ class CkeditorPreparer implements EditorPreparer {
     this.initCkeditor(dom, editCallback);
   }
 
-  notifySave(dom:DomContentBlock, success:boolean):void {
+  notifySave(dom:DomContentBlock, success:boolean, response:any):void {
     const cssClass = success ? 'success' : 'error';
     jQuery(dom.element).addClass(`renuo-cms-edit-${cssClass}`).delay(2000).queue(() =>
       jQuery(dom.element).removeClass(`renuo-cms-edit-${cssClass}`).dequeue());
+    if (!success) {
+      this.showErrorMessage(response.status);
+    }
+  }
+
+  private showErrorMessage(status:number):void {
```

Comment:
for me this is ok for now, i think we can refactor that to a service as soon as we display more than one message or want to rewrite to display a message-box with a div.